scaleway/data_source_registry_namespace_beta.go

```diff
@@ -0,0 +1,88 @@
+package scaleway
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/registry/v1"
+)
+
+func dataSourceScalewayRegistryNamespace() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceScalewayRegistryNamespaceRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "The name of the Registry Namespace",
+				ConflictsWith: []string{"namespace_id"},
+			},
+			"namespace_id": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Description:   "The ID of the Registry Namespace",
+				ConflictsWith: []string{"name"},
+				ValidateFunc:  validationUUIDorUUIDWithLocality(),
+			},
+			"region":          regionSchema(),
+			"organization_id": organizationIDSchema(),
+			"id": {
```

Comment:
do we need this ? 

it's missing from `resourceScalewayInstanceSecurityGroup`

Reply:
Yeah I think `id` is set by default (but here we do not inherit from another schema). May need confirmatin though :sweat_smile:

Reply:
Maybe add a test

Try it without this field

If it fails, try to add back this field.

Reply:
Works without :)